src/RightsService.Data/RoleRepository.cs

```diff
@@ -56,6 +56,11 @@ select new
          }).FirstOrDefault();
     }
 
+    public async Task<List<DbRole>> GetAllWithRightsAsync()
+    {
+      return await _provider.Roles.Include(role => role.RoleRights).ToListAsync();
```

Comment:
don't you want to check is active?

Reply:
I think no, because if we will check only roles with isActive status there could be 2 roles with the same rights set but with different statuses

Reply:
will check it in validator